opm/parser/share/keywords/000_Eclipse100/O/OPERNUM

```diff
@@ -0,0 +1,5 @@
+{
+ "name" : "OPERNUM",
```

Comment:
Again, I'm not sure what you mean.  The parser doesn't support `OPERNUM` either (read the manual).

Do you mean this?
https://github.com/OPM/opm-parser/blob/master/opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp#L197